libs/dh/market-participant/actors/feature-actors/src/lib/drawer/balance-responsible-relation-tab/dh-balance-responsible-relation-tab.component.html

```diff
@@ -0,0 +1,65 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<vater-flex
+  fill="horizontal"
+  justify="center"
+  align="flex-start"
+  *transloco="
+    let t;
+    read: 'marketParticipant.actorsOverview.drawer.tabs.balanceResponsibleRelation'
+  "
+>
+  <form
+    vater-stack
+    direction="row"
+    gap="m"
+    class="watt-space-stack-m"
+    tabindex="-1"
+    fill="horizontal"
+    [formGroup]="filterForm"
+  >
+    <watt-dropdown
+      [placeholder]="t('status')"
+      [chipMode]="true"
+      [options]="statusOptions"
+      [formControl]="filterForm.controls.status"
+    />
+    <watt-dropdown
+      [placeholder]="t('energySupplier')"
+      [chipMode]="true"
+      [options]="energySupplierOptions"
+      [formControl]="filterForm.controls.energySupplier"
+    />
+    <watt-dropdown
+      [placeholder]="t('gridArea')"
+      [chipMode]="true"
+      [options]="gridAreaOptions"
+      [formControl]="filterForm.controls.gridArea"
+    />
+    <watt-search #searchComponent [label]="t('search')" (search)="(searchEvent)" />
```

Comment:
suggestion: maybe something like this?

```suggestion
    <watt-search #searchComponent [label]="t('search')" (search)="searchEvent.emit($event)" />
```